src/main/java/HW_19/DataBaseConnection.java

```diff
@@ -0,0 +1,30 @@
+package HW_19;
+
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DataBaseConnection extends FileLoderProperties{
```

Comment:
Why did you extend FileLoderProperties?

Reply:
Please, remove target folder from PR

Reply:
remove target and delete extend FileLoderProperties